### PR TITLE
feat(execution-engine): Stream Map initial support [fixes VM-283, VM-284]

### PR DESCRIPTION
**Pull request by raftedproc**
feat(execution-engine): Stream Map initial support [fixes VM-283, VM-284]

### CODE REVIEW

File: air/src/execution_step/boxed_value/jvaluable/stream_map.rs [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2021 Fluence Labs Limited
```
Comment (monoid) [inline air/src/execution_step/boxed_value/jvaluable/stream_map.rs:2]: 2023

File: air/src/execution_step/execution_context/stream_maps_variables.rs [hunk 2]
```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright 2021 Fluence Labs Limited
```
Comment (monoid) [inline air/src/execution_step/execution_context/stream_maps_variables.rs:2]: 2023

File: air/src/execution_step/boxed_value/jvaluable/stream_map.rs [hunk 3]
```diff
@@ -0,0 +1,83 @@
+/*
```
Comment (raftedproc) [inline air/src/execution_step/boxed_value/jvaluable/stream_map.rs:1]: @mikevoronov Do you think I need this file at all?

File: air/src/execution_step/execution_context/stream_maps_variables.rs (hunk 2 above)
Comment (raftedproc) [reply air/src/execution_step/execution_context/stream_maps_variables.rs]: It was draft yet ) I fixed this.

**Pull request closed without merging**